app/Utils/ScSessionBookingInfo.cs

```diff
@@ -20,6 +20,10 @@ public class ScSessionBookingInfo
         public int ContainerId { get; set; }
         public int HearingTypeId { get; set; }
         public string HearingTypeName { get; set; }
+        public int EstimatedTrialLength { get; set; }
+        public bool IsHomeRegistry { get; set; }
+        public bool IsDifferentPlaceOfTrial { get; set; }
```

Comment:
IsDifferentPlaceOfTrial is kind of a confusing name because it sounds like it's just the negative of IsHomeRegistry.  

The actual question is "Do you have an entered court order changing the place of trial or is a different place of                            trial stated on the initiating documents (e.g. Notice of Civil Claim, Notice of Family Claim)?"

Can we call it something like **IsLocationChangeFiled** instead?

Reply:
That's a good suggestion, I originally had it as something like `differentPlaceDocuments` but then I wasn't sure if that was accurate and I left it at the time while I was getting the rest of it working. I'll make that update 👍